### PR TITLE
IOTEDGE-1024 Azure IoT Hub integration with FR Platform

### PR DESCRIPTION
**Pull request by JacoJooste**
Split up in two commits to show the changes required on top of the base IoT CDK configuration, which are:
- ICF Connector for IDM to query the devices on an Azure IoT Hub
- Configuration and scripts to support the integration

Documentation to follow.

### CODE REVIEW

File: examples/proof-of-concept/azure-iot/iot-hub-connector/pom.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Copyright 2016-2020 ForgeRock AS. All Rights Reserved
```
Comment (LimaEchoCharlie) [inline examples/proof-of-concept/azure-iot/iot-hub-connector/pom.xml:4]: Is this the correct licence?

File: examples/proof-of-concept/azure-iot/iot-hub-connector/src/main/java/org/forgerock/openicf/connectors/azure/AzureIotHubConnector.java [hunk 2]
```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2016-2017 ForgeRock AS. All Rights Reserved
```
Comment (LimaEchoCharlie) [inline examples/proof-of-concept/azure-iot/iot-hub-connector/src/main/java/org/forgerock/openicf/connectors/azure/AzureIotHubConnector.java:2]: 2017 -> 2020

**Review by LimaEchoCharlie — APPROVED** [review]
Awesome work Jaco

File: examples/proof-of-concept/azure-iot/iot-hub-connector/pom.xml (hunk 1 above)
Comment (JacoJooste) [reply examples/proof-of-concept/azure-iot/iot-hub-connector/pom.xml:4]: The files are all created/added by a Maven archetype and this is the license that they are created with. I agree that it's not very useful, but I can also not change them.

**Pull request merged**